system/include/emscripten/xmmintrin.h

```diff
@@ -483,6 +508,9 @@ _mm_cmpnlt_ss(__m128 __a, __m128 __b)
 static __inline__ int __attribute__((__always_inline__))
 _mm_comieq_ss(__m128 __a, __m128 __b)
 {
+#ifdef __EMSCRIPTEN__
+  if (isnanf(__a[0])) return 1;
+#endif
```

Comment:
Why is this code under ifdef __EMSCRIPTEN__? If this function needs to return true when the LHS is NaN, it would need to do that for any compiler because == doesn't do that.